extensions/notebook-renderers/src/test/notebookRenderer.test.ts

```diff
@@ -146,7 +146,7 @@ suite('Notebook builtin output renderer', () => {
 
 			const inserted = outputElement.firstChild as HTMLElement;
 			assert.ok(inserted, `nothing appended to output element: ${outputElement.innerHTML}`);
-			assert.ok(!outputElement.classList.contains('remove-padding'), `Padding should not be removed for non-scrollable outputs: ${outputElement.classList}`);
+			assert.ok(outputElement.classList.contains('remove-padding'), `Padding should be removed for non-scrollable outputs: ${outputElement.classList}`);
```

Comment:
unrelated, but this was changed a while ago, breaking this test (these tests don't run as part of CI yet)